web/public/js/controllers/dojos-map-controller.js

```diff
@@ -43,11 +43,27 @@ function cdDojosMapCtrl($scope, $window, $state, $stateParams, $translate, $geol
     });
 
     if($scope.model.map) {
-      $scope.model.map.setZoom(2);
-      $scope.model.map.setCenter(centerLocation);
       $scope.searchResult = null;
       delete $scope.search.dojo;
     }
+
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(function(position) {
+        var pos = {
+          lat: position.coords.latitude,
+          lng: position.coords.longitude
+        };
+        $scope.model.map.setCenter(pos);
+        $scope.model.map.setZoom(6);
```

Comment:
Should we perhaps increase the zoom level some more? I like 11. It seems to be close enough to get a good view of the surrounding area, and not too much pointless data further out.